share/completions/composer.fish

```diff
@@ -20,12 +20,26 @@ function __fish_composer_using_command
   return 1
 end
 
+function __fish_composer_packages
+    echo "
+import json
+json_data = open('composer.json')
```

Comment:
The `composer` command only works when you are in the directory with `composer.json`.  So nobody would usually start it from anywhere else *most likely*.  But if they did the code above would probably fail.

I will add some error handling.